### PR TITLE
SAIC-360 test for user migration with non-existing tenant

### PR DESCRIPTION
**Pull request by ntymtsiv**

### CODE REVIEW

**Comment by ntymtsiv** [comment]
Now this test failed due SAIC-359

File: devlab/tests/generate_load.py [hunk 1]
```diff
@@ -296,7 +296,7 @@ def create_all_networking(self):
                       if 'router:external' in net and net['router:external']
                       ][0]
         for tenant in config.tenants:
-            if tenant['networks']:
+            if 'networks' in tenant and tenant['networks']:
```
Comment (roman-verchikov) [inline devlab/tests/generate_load.py]: `tenant.get('networks')` should do the trick faster ;)

File: devlab/tests/generate_load.py [hunk 2]
```diff
@@ -426,6 +426,12 @@ def modify_admin_tenant_quotas(self):
                     'admin'), **tenant['quota'])
                 break
 
+    def delete_tenants(self):
+        for tenant in config.tenants:
+            if 'deleted' in tenant and tenant['deleted']:
```
Comment (roman-verchikov) [inline devlab/tests/generate_load.py]: same as above

**Comment by roman-verchikov** [comment]
Looks good other than two inlines, please fix

**Comment by roman-verchikov** [comment]
please rebase

**Comment by roman-verchikov** [comment]
+1

**Comment by realsystem** [comment]
+1

**Comment by roman-verchikov** [comment]
+1

**Pull request merged**